pkg/controllers/status/cluster_status_controller.go

```diff
@@ -67,6 +67,10 @@ const (
 	clusterNotReachableReason = "ClusterNotReachable"
 	clusterNotReachableMsg    = "cluster is not reachable"
 	statusCollectionFailed    = "StatusCollectionFailed"
+
+	apiEnablementSyncSucceed           = "SyncSucceed"
+	apiEnablementPartialAPIEnablements = "PartialAPIEnablements"
+	apiEnablementEmptyAPIEnablements   = "EmptyAPIEnablements"
```

Comment:
```suggestion
	apiEnablementsComplete = "CompleteAPIEnablements"
	apiEnablementsPartial  = "PartialAPIEnablements"
	apiEnablementsEmpty    = "EmptyAPIEnablements"
```
How about this? Just feel `SyncSucceed` not so straightforward.